daisy_workflows/export/disk_export.wf.json

```diff
@@ -14,7 +14,7 @@
       "Description": "list of GCE licenses to record in the exported image"
     },
     "export_instance_disk_image": {
-      "Value": "projects/compute-image-tools/global/images/family/debian-10-worker",
+      "Value": "projects/compute-image-tools/global/images/family/debian-11-worker",
```

Comment:
Why do you need to migrate from debian-10 to debian-11? Because of sfdisk?

Reply:
the reason is because the mounting of the disk runs into a bug with a debian-10 worker, but does not run into a bug with the debian-11 worker.